contributing.md

```diff
@@ -1,57 +1,148 @@
 
-Contributing
-============
+# Contributing
+
+
+## Getting Started
+
+All users wishing to contribute to OpenOA, or modify the code in any way, even for personal use,
+should follow much of the below guides.
```

Comment:
I suggest rephrasing this to be more declarative and less instructive:

"These contributing guidelines should be read by software developers wishing to contribute code or documentation changes into OpenOA, or to push changes upstream to the main NREL/OpenOA repository."